vendor/github.com/containers/image/docs/atomic-signature-embedded-json.json

```diff
@@ -1,66 +0,0 @@
-{
```

Comment:
Why does this PR remove the various files in `vendor`?

(Guessing, run `make vendor` instead of `vndr` directly.)